dispander/module.py

```diff
@@ -24,6 +24,14 @@ async def dispand(message):
     for m in messages:
         if m.content:
             await message.channel.send(embed=compose_embed(m))
+        if len(m.attachments) >= 2:
+            # Send the second and subsequent attachments with embed (named 'embed') respectively:
+            for attachment in m.attachments[1:]:
+                embed = Embed()
+                embed.set_image(
+                    url=attachment.proxy_url
+                )
+                await message.channel.send(embed=embed)
```

Comment:
多分 if 文要らないかな

[1:] でリストの2要素目以降を取得する場合、
なければ空リストが取得されてループ回数が0になるので、
これで良いはず

```suggestion
        # Send the second and subsequent attachments with embed (named 'embed') respectively:
        for attachment in m.attachments[1:]:
            embed = Embed()
            embed.set_image(
                url=attachment.proxy_url
            )
            await message.channel.send(embed=embed)
```

Reply:
確かにそうですね
最初popでやってたのでIndexError回避のために分岐してました
修正感謝です